tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -191,9 +191,9 @@ protected GraphQLTests(string sampleAppName, ITestOutputHelper output, string te
         {
             SetServiceVersion(ServiceVersion);
             SetEnvironmentVariable("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", metadataSchemaVersion);
-#if NETCOREAPP2_1
-            // NET 2.1, sometimes times out
-            SetEnvironmentVariable(Configuration.ConfigurationKeys.ObfuscationQueryStringRegexTimeout, "600");
+#if NETCOREAPP2_1_OR_GREATER
```

Comment:
nit but can probably just remove this `#if`

Reply:
+1 to this

Reply:
Makes sense, wasn't sure if taking it out was the best move, thank you.